plugin.cpp

```diff
@@ -1,3 +1,9 @@
+/*
+ * Copyright (C) 2016, Pelagicore AB.
+ * Author: Jonatan Pålsson <jonatan.palsson@pelagicore.com>
+ * SPDX-License-Identify: MPL-2.0
+ */
```

Comment:
I will open a discussion on the template file on GitHub, and we can discuss how it should look in that thread.